Doppler.HtmlEditorApi/Storage.DapperProvider/DapperFieldsRepository.cs

```diff
@@ -31,4 +31,10 @@ public async Task<IEnumerable<Field>> GetCustomFields(string accountName)
                 id: x.IdField,
                 name: x.Name,
                 isBasic: x.IsBasicField));
+
+    public async Task SaveFieldsId(int ContentId, IEnumerable<int> fieldsId)
+    => await new SaveFieldsId(_dbContext)
+            .ExecuteAsync(
+                new SaveFieldsId.Parameters(ContentId, fieldsId)
+            );
```

Comment:
Muy menor, pero por ahí estaría bien cambiar el formato del código para que encaje mejor con los otros métodos

```suggestion
    public async Task SaveFieldsId(int ContentId, IEnumerable<int> fieldsId)
        => await new SaveFieldsId(_dbContext)
            .ExecuteAsync(
                new SaveFieldsId.Parameters(ContentId, fieldsId));
```

Reply:
🚀